api/config/prest.toml

```
@@ -0,0 +1,10 @@
+[access]
+restrict = true
+
+[[access.tables]]
+name = "cnpj"
+permissions = ["read"]
+
+[[access.tables]]
+name = "meta"
+permissions = ["write"]
```

A API web não precisa escrever nada no banco, então acho que podemos restringir aqui tb, não?

```suggestion
permissions = ["read"]
```